pipeline/tests/asm.rs

```diff
@@ -22,7 +22,8 @@ fn slice_to_vec<T: FieldElement>(arr: &[i32]) -> Vec<T> {
 fn sqrt_asm() {
     let f = "asm/sqrt.asm";
     let i = [3];
-    let pipeline: Pipeline<GoldilocksField> = make_prepared_pipeline(f, slice_to_vec(&i), vec![]);
+    let pipeline: Pipeline<GoldilocksField> =
+        make_prepared_pipeline(f, slice_to_vec(&i), vec![], LinkerMode::Bus);
```

Comment:
So these are only testing with bus mode?

Reply:
Yes, I think it should be the default now, whenever possible.